### PR TITLE
Profile command

### PR DESCRIPTION
**Pull request by davipatury**
profile
canary-profile

### CODE REVIEW

**Comment by davipatury** [comment]
# READY FOR REVIEW

**Comment by pedrofracassi** [comment]
One last thing: Can the gradient use the `EMBED_COLOR` env var?

File: src/commands/social/profile.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      new UserParameter({full: true, required: false})
+    )
+  }
+
+  async run ({ t, author, channel, client }, user) {
+    user = user || author
+    channel.startTyping()
+    const before = Date.now()
+    const profile = await CanvasTemplates.profile({ t, client }, user)
+    channel.send(`Took **${Date.now() - before}**ms`, new Attachment(profile, 'profile.jpg')).then(() => channel.stopTyping())
```
Comment (bolsomito) [inline src/commands/social/profile.js]: move the strings to `strings.json`

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -246,5 +246,9 @@
   "imageoftheday": {
     "commandDescription": "Shows the NASA Image of the Day",
     "embedAuthor": "NASA Image of the Day"
+  },
+  "profile": {
+    "commandDescription": "Shows your or someone else's profile",
+    "commandUsage": "<user>"
```
Comment (bolsomito) [inline src/locales/en-US/commands.json]: [user] as it isn't obligatory

File: src/commands/social/profile.js (hunk 1 above)
Comment (pedrofracassi) [reply src/commands/social/profile.js]: The only thing that needs to be moved is "Next reward in". "Switchcoins" is already a string in `commons.json`.

**Comment by pedrofracassi** [comment]
I think we should add profile customization commands before publishing.

File: src/commands/social/profile.js (hunk 1 above)
Comment (bolsomito) [reply src/commands/social/profile.js]: not that dummy, i'm talking about "Took **{{time}}**ms"

Comment (pedrofracassi) [reply src/commands/social/profile.js]: That's temporary

**Comment by bolsomito** [comment]
url?

**Comment by davipatury** [comment]
This PR will be merged into `dev` branch, we can add the profile customization commands in another PR.

**Comment by pedrofracassi** [comment]
Great!

**Comment by Doges** [comment]
then merge it :^)

**Pull request merged**